test/runtests.jl

```diff
@@ -1,66 +1,85 @@
 using NIDAQ, Base.Test
 
+info("NIDAQ: Checking installation - no measurement hardware is needed for this section")
 @test typeof(getproperties()) == Dict{ASCIIString,Tuple{Any,Bool}}
-d = devices()[1]
-@test typeof(getproperties(d)) == Dict{ASCIIString,Tuple{Any,Bool}}
-@test typeof(analog_input_channels()) == Vector{ASCIIString}
-@test typeof(analog_output_channels()) == Vector{ASCIIString}
-@test typeof(digital_input_channels()) == Vector{ASCIIString}
-@test typeof(digital_output_channels()) == Vector{ASCIIString}
-@test typeof(counter_input_channels()) == Vector{ASCIIString}
-@test typeof(counter_output_channels()) == Vector{ASCIIString}
-@test typeof(analog_input_ranges()) == Matrix{Float64}
-@test typeof(analog_output_ranges()) == Matrix{Float64}
```

Comment:
please retain the above 8 lines.  they might need to be moved inside the `if device_available` logic